project.clj

```diff
@@ -31,12 +31,8 @@
   :profiles {:dev {:plugins      [[com.jakemccrary/lein-test-refresh "0.24.1"]
                                   [lein-ancient "0.6.15"]
                                   [lein-auto "0.1.3"]
-                                  [lein-set-version "0.4.1"]
-                                  [venantius/ultra "0.6.0"]]
```

Comment:
Ultra should be installed per user.

https://github.com/venantius/ultra#installation

Reply:
Also seeing issues with latest clojure.

https://github.com/venantius/ultra/issues/108

Reply:
Breaks new IntelliJ (2019.2) with Cursive support as well

Reply:
Ultra also does dependency re-writing for the project.clj..